modules/031-linstor/docs/CONFIGURATION_RU.md

```diff
@@ -98,3 +98,249 @@ data:
 Для решения этой задачи укажите лейбл `linstor.csi.linbit.com/on-storage-lost: remove` в описании Pod'а. Модуль linstor автоматически удалит такие Pod'ы с узла где возникли проблемы, что позволит Kubernetes перезапустить приложение на другом узле.
 
 [Пример...](usage.html#перенос-приложения-при-проблемах-с-узлом-storage-based-fencing)
+
+## Резервное копирование в S3
+
+Резервное копирование данных реализовано с помощью [снапшотов томов](https://kubernetes.io/docs/concepts/storage/volume-snapshots/). Поддержка работы снапшотов обеспечивается модулем [snapshot-controller](../045-snapshot-controller/), который включается автоматически для поддерживаемых CSI-драйверов в кластерах Kubernetes версий 1.20 и выше.
+
+> Возможна работа не со всеми S3-совместимыми сервисами. Подтверждена работа с [Amazon S3](https://aws.amazon.com/s3/) и [MinIO](https://min.io).
+>
```

Comment:
```suggestion
```

fixed in [v1.19.0](https://github.com/LINBIT/linstor-server/releases/tag/v1.19.0)